docs/releases.md

```diff
@@ -0,0 +1,127 @@
+
+# Release Documentation
+
+# Abstract
+
+In order to provide a generic guidance to users, downstream maintainers and
+any other consumers of the Cloud Hypervisor project, this document
+describes the release process, release cadence, stability expectations and
+related topics.
+
+# Basic Terms
+
+## Stability
+
+In the context of Cloud Hypervisor, the points of reference regarding stability
+are as below:
+
+- [REST API](api.md#rest-api).
+- [Command line options](api.md#command-line-interface).
+- [Device Model](device_model.md).
+- Device tree, device list, ACPI, Hyper-V enlightenments and any other features
+  exposed to guest.
+- KVM compatibility.
+- UEFI compatibility.
```

Comment:
Again, I don't see how that is different to the interface exposed to the guest you mention above. Cloud Hypervisor has nothing to to with UEFI - that is an abstraction provided by a firmware to payload of said firmware.

Reply:
That's correct. The purpose of mentioning it separately is that UEFI doesn't belong to the CH repo, while it is essential and indeed maintained by the CH org. Besides the services provided to the guest, the firmware itself is to be of a certain layout. Say even if same services were present in a QEMU or any other firmware build, at the CH level it can be already refused (that's the case on Aarch64 for sure, on Intel compatible arch OVMF might be better interchangeable).

I'd then just put this as a single mention into the above list, so then it's clear to be something considered as a stability factor, too.

thanks

Reply:
Changed that way. Thanks